src/provider/model/Toast.ts

```diff
@@ -266,17 +266,13 @@ export class Toast implements LayoutItem {
         Object.assign(this._origin, transform.origin);
 
         // TODO [SERVICE-739] This is a fix for setBounds not working on hidden windows.
-        await (await this._webWindow).showAt(transform.origin.x, transform.origin.y);
-
-        await this.trackAnimation((await this._webWindow).setBounds({
+        const track = Promise.all([(await this._webWindow).showAt(transform.origin.x, transform.origin.y), (await this._webWindow).setBounds({
```

Comment:
Is this definitely not a race condition or anything? From how you explained the problem, `setBounds` would fail if the window wasn't visible, but this won't wait for the `showAt` to finish before the `setBounds` starts.

Something like `trackAnimation(showAt.then(setBounds))` should fix both issues..?